src/ImpactX.cpp

```diff
@@ -279,6 +285,9 @@ namespace impactx
               py.reserve(npart);
               pt.reserve(npart);
 
+              // write file header
+              amrex::PrintToFile("initial_beam.txt") << "#x y t px py pt\n";
```

Comment:
@WeiqunZhang how do people handle multiple outputs between runs. I think I saw a standard renaming scheme that we use to move old files out?

Currently, this will append to a previous run's output.

Reply:
Old files can be removed with `amrex::FileSystem::Remove(std::string const& filename)`.

Reply:
Thx! Ok that looks a bit tricky, because we suffix the files in `amrex::PrintToFile` automatically with MPI rank and OMP thread

Reply:
Found a solution: #65 :)